go/cmd/dolt/commands/log.go

```diff
@@ -99,6 +104,7 @@ func (cmd LogCmd) ArgParser() *argparser.ArgParser {
 	ap.SupportsFlag(parentsParam, "", "Shows all parents of each commit in the log.")
 	ap.SupportsString(decorateParam, "", "decorate_fmt", "Shows refs next to commits. Valid options are short, full, no, and auto")
 	ap.SupportsFlag(oneLineParam, "", "Shows logs in a compact format.")
+	ap.SupportsString(notParam, "", "revision", "Reverses the meaning of the ^ prefix (or lack thereof) for all following revision specifiers, up to the next --not.")
```

Comment:
The `--not` param feels a little confusing. I see that this is 100% just coming from how Git uses it; it just feels kinda harder to think about for me than the '^' sign. It seems weird that it reverses all following params until the next `--not` is seen, but I get that we're just following Git. It doesn't look like we can support multiple `--not`s though? So maybe we could simplify the docs a little bit knowing that? Might also want to add a BATS test for `--not` reversing multiple params just so we've got that case covered, too. Or, if you're also feeling weird about `--not` we could omit it for now.

Reply:
Changed the copy here/more bats so that it's clearer we only support one `--not`